File: sdkwork-app-sdk-csharp/Models/VipCouponVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VipCouponVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Name { get; set; }
        public double? Discount { get; set; }
        public double? MinAmount { get; set; }
        public double? ReduceAmount { get; set; }
        public string? ValidFrom { get; set; }
        public string? ValidTo { get; set; }
        public string? Status { get; set; }
        public bool? Used { get; set; }
    }
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/VipCouponVO.kt
package com.sdkwork.app

data class VipCouponVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: String? = null,
    val name: String? = null,
    val discount: Double? = null,
    val minAmount: Double? = null,
    val reduceAmount: Double? = null,
    val validFrom: String? = null,
    val validTo: String? = null,
    val status: String? = null,
    val used: Boolean? = null
)
